### DISCUSSION TITLE
Close currently opend dialog from anywhere.

### DISCUSSION THREAD

**Discussion by derech1e** [canton7/Stylet#350] [Q&A] [answered]
Is there a way to close a currently opend dialog from a other location without having direct access to the instance of the dialog?

**Comment by canton7** [comment, marked as answer]
You need some way of telling the dialog you want it to close. You could keep a reference to it, or send it a message through the Event Aggregator, or store the instance somewhere (e.g. make a service which manages that dialog, and register it as a singleton with the ioc container).
